docs/round_to_fraction.rst

```diff
@@ -0,0 +1,124 @@
+df.round_to_fraction()
```

Comment:
The examples guides have been great! I'm wondering about how it will render on readthedocs, and the maintainability of two separate files (one Markdown, one RST). Have you fiddled with sphinx before? Readthedocs uses Sphinx to build documentation, and you can build it locally too. In that way, there's fewer duplicate things to maintain.

Reply:
Sure, I'll work with that. I'm just more used to Markdown (since you can even use it in these textboxes), but RST is ok too.